recipes/deepchecks/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "deepchecks" %}
+{% set version = "0.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/deepchecks-{{ version }}.tar.gz
+  sha256: ee6049bfba2bb0e863d027756c2eb29ae2b10285ac7ab7eee3cc0d8e0c9e6a58
+
+build:
+  number: 0
+  # noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - category_encoders >=2.3.0
+    - dataclasses >=0.6
+    - ipykernel >=4.10.1
+    - ipykernel >=5.3.0  # [ py >= 38 ]
```

Comment:
is this selector really necessary?

Reply:
I stuck to the recipe definition the package-maintainers are using for their own channel (`deepchecks`). This way, it will be easier for maintenance.

The only reason why I did not remove it is the are using different dependency versions for different python versions. And I am not sure if there is a standard way of doing that in conda-forge (apart from what the recipe has now).

Reply:
no there is not. the big downside is that the recipe cannot be `noarch` which is quite unfortunate -- if it was noarch it would be immediately available on all platforms etc.
I don't really see how these extra dependencies for `py >= 38` make sense. One could also require `python >=3.8` for the entirety of the recipe?

Reply:
From my point of view, there is not much of a sense to have that selector, especially if you render this recipe for python 3.8 you will have two dependencies specifications for the same package

Reply:
the same discussion is valid for ``ipywidgets``

Reply:
So, what do you think would be an optimal compromise?

```diff
  	  - ipykernel >=4.10.1
-     - ipykernel >=5.3.0  # [ py >= 38 ]
```

Reply:
Changed it into a noarch package as suggested.